src/Bladeburner/Bladeburner.ts

```diff
@@ -750,32 +751,20 @@ export class Bladeburner {
     }
   }
 
+  public killRandomSupportingSleeves(n: number) {
```

Comment:
Optional: This function doesn't depend on *any* data in Bladeburner. (I.e. no `this` or anything like that.) Does it need to be part of Bladeburner? Does somewhere else make more sense?

Reply:
Hm.... Yes it needs to be here. It's hard to spot but the coupling is **efferent** (going out) to the _Player_ global static. Should we at some point resolve the usage of the static global we'd have to inject the PlayerObject explicitly (not the person performing the action, as sleeve actions cannot have a teamCount > 0). There isn't a simple way of handling that right now without making a mess, but moving this out of here will make it more difficult to do so.

I have a PR for completeAction test/refactor lined up that does address this to some degree.